### PR TITLE
manifest: nrf_modem: version 1.2.3

### PR DESCRIPTION
**Pull request by MirkoCovizzi**
See the CHANGELOG for a complete list of changes.

Signed-off-by: Mirko Covizzi <mirko.covizzi@nordicsemi.no>

### CODE REVIEW

**Comment by richardmccrae-nordicsemi** [comment]
This PR has been tested with all libmodem system tests using the test repo branch that matches NCS 1.6.0. Everything looks good
[libmodem_1.2.3_results.pdf](https://github.com/nrfconnect/sdk-nrf/files/9615482/libmodem_1.2.3_results.pdf)

**Review by richardmccrae-nordicsemi — APPROVED** [review]

**Review by eivindj-nordic — APPROVED** [review]
LGTM

**Review by lemrey — APPROVED** [review]
@divipillai are there requirements on the changelog for this PR, given that it is being merged to `v1.6-branch` ?

**Comment by divipillai** [comment]
> @divipillai are there requirements on the changelog for this PR, given that it is being merged to `v1.6-branch` ?

We don't need a change log entry. But we need to have an entry in the known issues for the NCS versions (other that v1.6) that still have this issue (That would be a separate PR).

**Comment by MirkoCovizzi** [comment]
@divipillai Is it possible to have an exhaustive list of the NCS versions that require an update to the known issues? Thanks

**Review by divipillai — APPROVED** [review]

**Comment by divipillai** [comment]
@gmarull : Could you please check this doc build Issue : https://github.com/nrfconnect/sdk-nrf/actions/runs/3127666446/jobs/5074556000

**Comment by gmarull** [comment]
> @gmarull : Could you please check this doc build Issue : [nrfconnect/sdk-nrf/actions/runs/3127666446/jobs/5074556000](https://github.com/nrfconnect/sdk-nrf/actions/runs/3127666446/jobs/5074556000)

please rebase, branch seems outdated (quite a lot btw) https://github.com/MirkoCovizzi/sdk-nrf/blob/v1.2.3/west.yml#L59

**Comment by divipillai** [comment]
@gmarull : Just FYI - This PR is to be merged to the 1.6 branch.

**Comment by gmarull** [comment]
> @gmarull : Just FYI - This PR is to be merged to the 1.6 branch.

Then just override CI, I think it's not worth investigating this issue since we do not render branch docs

File: west.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: v1.6.1
+      revision: pull/831/head
```
Comment (rlubos) [inline west.yml:117]: Please update SHA: `932455ada40b103b04de9d036d35b2c4043f9bbf`

**Review by rlubos — APPROVED** [review]

**Pull request merged**